### ISSUE TITLE
Exceptions do not work

### ISSUE THREAD

**Issue by Nywuob** [beezee/djax#16]
I want to exclude some links to images that should be opened with Fancybox, but they still get the "dJax_internal" class.

This is the code i'm using:

```
jQuery(document).ready(function($) {
    $('body').djax('.updatable', ['assets']);
});
```

Example image link:

<pre>
a href="assets/templates/wwaw/images/1_b.jpg" class="fancybox" data-fancybox-group="gallery" title="Lorem ipsum dolor sit amet">
</pre>

Is this a bug or am I doing something wrong?

**Comment by beezee** [comment]
Is this the case in all browsers? I personally haven't had trouble with this, but I'm using Javascripts indexOf method, might not have full coverage. Lemme know if there's a browser specific issue and I'll switch the method for detecting exceptions if need be.

**Comment by Nywuob** [comment]
Yes, it happens in the latest versions of Chrome, Firefox and Opera. I don't know about safari, but I reckon it's the same.

Thanks for your quick response btw.

**Comment by Nywuob** [comment]
Any updates on this issue?

**Comment by beezee** [comment]
Unfortunately I've not been able to reproduce- if you can host a simple demonstration of the issue I'll be happy to dig into the source
